kafka-rest/src/main/java/io/confluent/kafkarest/backends/kafka/KafkaModule.java

```diff
@@ -33,35 +37,48 @@
  *
  * <p>Right now this module does little but delegate to {@link KafkaRestContext}, since access to
  * Kafka is currently being configured there. It's the author's intention to move such logic here,
- * and eliminate {@code KafkaRestContext}, once dependence injection is properly used elsewhere.</p>
+ * and eliminate {@code KafkaRestContext}, once dependence injection is properly used elsewhere.
```

Comment:
the style checker was not unhappy about this?

Reply:
Google Java Style forbids the closing `</p>`. This has been auto-formatted. Check-style is fine with it, since it's optional.